docker-compose.test.yml

```diff
@@ -100,6 +100,7 @@ services:
 
   db-ja:
     image: mariadb:latest
+    hostname: ${RANDOMTAG}db-ja
```

Comment:
If you add the randomtag here then it also has to be added in the DATABSE_URL from the jitsi-admin service.
But this bug is correct and thanks for fixing this issue